go.mod

```diff
@@ -1,50 +1,50 @@
 module github.com/dundee/gdu/v5
 
-go 1.22
+go 1.24.4
```

Comment:
I'm afraid we cannot bump this yet. 

I would like to keep support for latest LTS version of Ubuntu - [currently 24.04 Noble Numbat](https://releases.ubuntu.com/noble/), which [uses Golang 1.22](https://packages.ubuntu.com/noble/golang).

Reply:
In that case we can go ahead and close out this PR?
There are a couple of "corrections" from my previous PR though. Those can be done later, I guess.